app/services/class_service.py

```diff
@@ -0,0 +1,65 @@
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+from app.models.class_model import Class
+from app.schemas.class_schema import ClassCreate, ClassUpdate
+
+
+def get_all_classes(db: Session, page: int, limit: int, name: str = None):
+    query = db.query(Class).filter(Class.disabled == False)
+    if name:
+        query = query.filter(Class.name.ilike(f"%{name}%"))
+    total = query.count()
+    classes = query.offset((page - 1) * limit).limit(limit).all()
```

Comment:
Get skip by using function from utils.

```suggestion
skip  = paginate.getSkip(page=page, limit=limit)
```